models/index.js

```diff
@@ -5,7 +5,8 @@ import dotenv from 'dotenv';
 import config from '../config/config';
 
 const basename = path.basename(__filename);
-const db = {};
+const db = {
+};
```

Comment:
Why are the curly braces on different lines?

Reply:
done